test/transaction.ts

```diff
@@ -56,8 +65,8 @@ class FakeDatastoreRequest {
 
 describe('Transaction', () => {
   // tslint:disable-next-line variable-name
-  let Transaction;
-  let transaction;
+  let Transaction: Any;
```

Comment:
Should be something like `: typeof tsTypes.Transaction`

Reply:
This one was getting me a bit - in case I'm not understanding you correctly, I wrote:
```
import * as tsTypes from '../src/transaction'
...
let Transaction: typeof tsTypes.Transaction;
let transaction: tsTypes.Transaction
```

Reply:
PERFECT.